CHANGES.md

```diff
@@ -4,7 +4,8 @@
 
 ### Fixes
 
-
+* xcube now takes care to avoid indexing timezone-naive variables with
```

Comment:
"Prevents" is a bit too strong (in part because we can't guarantee that every datetime-like value will support naive/aware interconversion), but I'll compromise on "tries to prevent".